content/04.network_applications.md

```diff
@@ -110,84 +110,85 @@ Categories consist of: (a) Multi-Omic applications, (b) Pharmaceutical Applicati
-High scores implicated high chance for an association [@doi:10.1093/jamia/ocy117].
+Applications that use deep learning techniques have mainly focused on the node2vec model [@arxiv:1607.00653] and other variants of the model.
+Yang et al. used node2vec to create a recommendation system to infer associations between genes and disease symptoms [@doi:10.1093/jamia/ocy117].
+The authors constructed a gene-disease symptom knowledge graph by combining two bipartite graphs: genes with diseases and diseases with disease symptoms.
+The generated graph was embedded via node2vec and similarity scores were calculated for every gene-symptom pair in the graph.
+High scores implicated high likelihood for an association [@doi:10.1093/jamia/ocy117].
 This approach outperformed methods that didn't use a knowledge graph; however, validation was difficult as it involved manual curation of the literature @doi:10.1093/jamia/ocy117].
-Similar approaches used variants of the node2vec algorithm to predict gene-disease associations
+Similar approaches used variants of the node2vec model to predict gene-disease associations
 [@doi:10.1093/bioinformatics/bty559; doi:10.3389/fgene.2019.00226 @doi:10.1186/s12920-019-0627-z; @doi:10.1109/BIBM47256.2019.8983134] analyze RNA-seq data [@doi:10.1093/nar/gkx750] and infer novel protein information [@doi:10.1093/bioinformatics/btx275; @doi:10.1007/978-3-030-00825-3_16; @doi:10.1016/j.artmed.2019.04.001; @doi:10.1186/s12859-018-2163-9].
-Future extensions of these applications should consider incorporating more sources of data such as compounds, anatomic locations or even gene pathways to improve prediciton performance.
+Future applications should consider incorporating more sources of data such as compounds, anatomic locations or even gene pathways to improve prediction performance.
 
-Knowledge graphs have aided the multi-omics field by generating novel discoveries.
-Most approaches to date use matrix factorization and node2vec to project knowledge graph into a low dimensional space, and translational models may be an untapped resource that could aid future efforts.
-Another area of exploration could be the incorporation of multiple sources of information such as anatomic locations or genetic pathways to improve the specificity of findings (i.e., to predict that a protein-protein interaction happens in a specific cell type or tissue).
+Knowledge graphs benefited the multi-omics field as a resource for generating novel discoveries.
+Most approaches to date use matrix factorization and node2vec to project knowledge graph into a low dimensional space, while translational models may be an untapped resource that could aid future efforts.
+Another area of exploration could be incorporating multiple sources of information such as anatomic locations or genetic pathways to improve the specificity of findings (i.e., to predict that a protein-protein interaction happens in a specific cell type or tissue).
```

Comment:
This paragraph and the one above seem to end with the same rough statement (or at least sentiment). Should the sentiments in the one above be moved down here?